tutorials/transform-bucket-images-triggers-functions-set-up/index.mdx

```diff
@@ -0,0 +1,132 @@
+
+Serverless Functions are an asynchronous microservices architecture where event sources are decorrelated from event consumers.
+
+They work best when they are triggered by events stored in queues, which means that they can be edited without having to modify every microservices in the process.
+
+In this tutorial, you will create the following resources to use your functions:
+
+- A project dedicated to this tutorial
+- Two buckets to store media
+- A queue to manage events
+- IAM credentials to secure your environment
+- Cockpit to monitor your functions
+
+<Macro id="iam-requirements" />
+ 
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com).
+  - You have around 10 JPG or PNG images.
+  - You have [Vim](https://www.vim.org/download.php) installed.
+</Message>
+
+## Creating a New Project
+
+1. Click the **Projects** tab from the [Organization Dashboard](https://console.scaleway.com/organization). A list of your Projects displays.
+2. Click **+ Create Project** on the righthand side. A pop-up appears. 
+3. Name the project "Functions Tutorial".
+4. Click **Create Project**. The new Project is added to the list. 
+
+## Creating the source bucket
+
+1. Click **Object Storage** under **Storage** on the left side menu of the console. The Object Storage dashboard displays.
+3. Click **+ Create a bucket**.
+4. Name it "source-images-<YOUR_NAME>".
+5. Chose the **AMSTERDAM** Region.
+6. Set the bucket visibility to **Private**.
+7. Click **Create Bucket**. The bucket **Files** tab displays.
+8. Upload your images to the bucket and select the **Standard** storage class.
+
+## Creating the destination bucket
+
+1. Click **Storage**, then **Object Storage** on the left side menu of the console. The Object Storage dashboard displays.
+2. Click **+ Create a bucket**.
+3. Name it "dest-images-<YOUR_NAME>".
+4. Chose the **AMSTERDAM** Region.
+5. Set the bucket visibility to **Private**.
+6. Click **Create Bucket**.
+
+## Creating a messaging and queuing namespace
+
+Message queues allow functions to communicate with each other.
+
+1. Click **Messaging** under **Serverless**.
+2. Click **Create namespace**.
+3. Choose **SQS/SNS** as type.
+4. Choose a name or generate one.
+5. Click **Create namespace**.
+
+## Generating credentials for the M&Q namespace
+
+1. Click on the namespace you just created.
+2. Click **Generate credentials**.
+3. Name your credentials "Function Push".
+4. Enable the **Write**, **Read** and **Manage** permissions.
+5. Click **Generate credentials**.
+6. Store the Access key ID and the Secret key securely, as you will not be able to access them later.
+
+## Creating a Queue
+
+1. Click the **Queues** tab of your namespace.
+2. Click **Create queue**.
+3. Select *Standard** as a queue type.
+4. Name it "image-queue". 
+5. Keep the default parameters.
+6. Click **Create queue**.
+
+## Creating IAM credentials
+
```

Comment:
same here, you can maybe just add a link to the procedure to make tuto a bit shorter, but it's ultimately up to you!

Reply:
We need to force some options (Set up preferred project for object storage, User as a bearer and 1 week expiration, name) for this tutorial.  IAM How-to leaves the choice open, which can lead to tutorial not working. Also, numbers of steps performed remains the same for the user, so might as well keep it now that it is written!